server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -25,7 +25,7 @@ export PB_ELASTICSEARCH_IMAGE_NAME=${PB_ELASTICSEARCH_IMAGE_NAME:-"elasticsearch
 export PB_ELASTICSEARCH_IMAGE_TAG=${PB_ELASTICSEARCH_IMAGE_TAG:-"7.13.3"}
 if [[ "${PB_CONTAINER_REG}" == "localhost" ]]; then
     # Drop the explict registry reference when using "localhost".
-    export PB_ELASTICSEARCH_IMAGE="${PB_ELASTICSEARCH_IMAGE_NAME}:${PB_ELASTICSEARCH_IMAGE_TAG}"
+    export PB_ELASTICSEARCH_IMAGE="${PB_CONTAINER_REG}/${PB_ELASTICSEARCH_IMAGE_NAME}:${PB_ELASTICSEARCH_IMAGE_TAG}"
```

Comment:
Tagging the changed line even though I'm commenting on the comment: you're not dropping the "explicit registry reference" now, but the org name.

Reply:
Perhaps Dave cannot, but I can:  🙂 
```
wscales@pbench-ctr-build pbench, Tue Jan 31 16:49:16
$ podman tag fc8fcabe6067 localhost/pbench/pbench-ci-fedora:localhost_tag_test
wscales@pbench-ctr-build pbench, Tue Jan 31 16:50:24
$ podman image ls | head
REPOSITORY                                               TAG                 IMAGE ID      CREATED        SIZE
localhost/pbench-ci-fedora                               main                c878297adff0  5 days ago     1.7 GB
localhost/pbench/pbench-ci-fedora                        localhost_tag_test  fc8fcabe6067  10 days ago    3.22 GB
quay.io/pbench/pbench-ci-fedora                          main                fc8fcabe6067  10 days ago    3.22 GB
```

AFAIK, the "org name" is not a formal concept in docker/podman -- it's just part of the repository name.